packages/itwinui-react/src/core/ThemeProvider/ThemeProvider.tsx

```diff
@@ -84,6 +86,16 @@ type ThemeProviderOwnProps = Pick<RootProps, 'theme'> & {
    * </ThemeProvider>
    */
   portalContainer?: HTMLElement;
+  /**
+   * This prop will be used to determine if `styles.css` should be automatically imported at runtime.
+   *
+   * When using `theme='inherit'`, styles will be automatically loaded if not already found.
+   * This default behavior is useful for packages that want to support incremental adoption of latest iTwinUI,
+   * without requiring consuming applications (that might still be using an older version) to manually import the CSS.
+   *
+   * If true or false is passed, it will override the default behavior.
+   */
+  includeCss?: boolean;
```

Comment:
nit: Clarity

```suggestion
  /**
   * This prop will be used to determine if `styles.css` should be automatically imported at runtime.
   *
   * By default, this is set to `true` if `theme='inherit'`. If `true`, styles will automatically be loaded, if not already found.
   * This default behavior is useful for packages that want to support incremental adoption of latest iTwinUI,
   * without requiring consuming applications (that might still be using an older version) to manually import the CSS.
   *
   * Passing `true` or `false` will override the default behavior.
   * 
   * @default
   * theme='inherit'
   */
  includeCss?: boolean;
```

Reply:
How about this?

I don't like specifying `@default` here since that is intended for simple values.

```suggestion
  /**
   * This prop will be used to determine if `styles.css` should be automatically imported at runtime (if not already found).
   *
   * By default, this is enabled when using `theme='inherit'`.
   * This default behavior is useful for packages that want to support incremental adoption of latest iTwinUI,
   * without requiring consuming applications (that might still be using an older version) to manually import the CSS.
   *
   * If true or false is passed, it will override the default behavior.
   */
  includeCss?: boolean;
```

Reply:
Ohh, yes, your concise version sounds good 👍

Reply:
pushed